crates/driver/src/tests/setup/mod.rs

```diff
@@ -128,8 +128,9 @@ pub struct Order {
     /// buy amount is divided depends on the order side. This is necessary to
     /// keep the solution scores positive.
     pub surplus_factor: eth::U256,
-    /// Override the executed amount of the order. Useful for testing liquidity
-    /// orders. Otherwise [`execution_diff`] is probably more suitable.
+    /// Override the executed target amount of the order. Useful for testing
+    /// liquidity orders. Otherwise [`execution_diff`] is probably more
+    /// suitable.
```

Comment:
You mean to pass order's full amount into this function?
https://github.com/cowprotocol/services/blob/2ac00e79749f67a2717207181a95bdb18d528873/crates/driver/src/tests/setup/blockchain.rs#L64

That doesn't work since `ab_adjusted_pool` function adjust one of the pool's reserves to align with the quote's ratio. For different inputs, the pool returns different results.